pandas/tests/arrays/sparse/test_array.py

```diff
@@ -1295,3 +1295,17 @@ def test_map_missing():
 
     result = arr.map({0: 10, 1: 11})
     tm.assert_sp_array_equal(result, expected)
+
+
+def test_dropna_sparse_column_doesnt_drop_nonna():
+    arr = SparseArray([np.nan, 1])
+    exp = SparseArray([1.0])
+    assert len(arr.dropna()) == 1
```

Comment:
might be a bit redundant? the line below (`tm.assert_sp_array_equal` will ensure len is 1 as well, right?)